functests/3_performance_status/status.go

```diff
@@ -139,3 +154,35 @@ func createBadMachineConfig(name string) *machineconfigv1.MachineConfig {
 		},
 	}
 }
+
+func createBadTuned(name, ns string) *tunedv1.Tuned {
+	priority := uint64(20)
+	data := "[main]\nsummary=A Tuned daemon profile that does not exist\ninclude=profile-does-not-exist"
+
+	return &tunedv1.Tuned{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: tunedv1.SchemeGroupVersion.String(),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: ns,
+			UID:       types.UID(utilrand.String(5)),
+		},
+		Spec: tunedv1.TunedSpec{
+			Profile: []tunedv1.TunedProfile{
+				{
+					Name: &name,
+					Data: &data,
+				},
+			},
+			Recommend: []tunedv1.TunedRecommend{
+				{
+					MachineConfigLabels: map[string]string{"machineconfiguration.openshift.io/role": testutils.RoleWorkerCNF},
```

Comment:
maybe we should explain the connection here (or in the test itself) . its not intuitive that creating a misconfigured tuned CR  with the same `machineconfiguration.openshift.io/role` will lead a degraded state of the tuned PAO creates from the performance profile

Reply:
Because it's not. the tuned that PAO created is perfectly fine.
During this test we're creating additional synthetic tuned CR by invoking the ```createrBadTuned``` function.
This synthetic tuned will look for a tuned profile which doesn't exist. 
see ```include=profile-does-not-exist``` line in [here](https://github.com/openshift-kni/performance-addon-operators/pull/598/files/747cbc5b937af9aded6afda395c8359c747db471#diff-46dd9c2fd680952d6ae2734f9f4b9d42734d386f897dcb79cf030f17eae6f8f4R160)

This tuned CR will be applied on the ```profiles.tuned.openshift.io``` CR (there is such profile per node) which is associate to the node object with the same name.
The connection between the node object and the tuned object is via the ```MachineConfigLables``` in our case ```worker-cnf``` 

The initial idea which was to modify the same tuned CR that PAO creates, turned out to be a bad idea, since the system not recover from such state.

Reply:
I'll comment the above explanation in the code